react/week4/my-app/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, createContext } from "react";
+import "./App.css";
+import Searcher from "./components/Searcher";
+import FetchUser from "./components/FetchUser";
+import Repos from "./components/RenderProps";
+
+export const stateContext = createContext();
```

Comment:
Contexts should be named starting from a capital letter by convention. Also, it's better to move it to a separate file, so you don't import `App.js` in all of the places you need access to `stateContext`